tools/Run.cmd

```diff
@@ -0,0 +1,69 @@
+@echo off
+
+rem |  Common pattern of MSBUILD target invocation
+rem |  for Pandell builds.
```

Comment:
Ditto, please add a note that this file comes from `node-repo-init` and what the version at the time of copy was.